src/main/groovy/com/wooga/gradle/PropertyLookup.groovy

```diff
@@ -163,22 +175,92 @@ class PropertyLookup {
         return false
     }
 
+    Integer getValueAsInteger(Map<String, ?> properties, Map<String, ?> environment = null) {
+        (Integer) getValue(properties, environment)
```

Comment:
I think a `Integer.parse` would be cleaner that to hope that a cast will do the trick.

Reply:
Sure, I can call parse after strinfiying the object.